testing-basics-garden-project/garden-test008.js

```diff
@@ -0,0 +1,17 @@
+import { Garden } from "../src/Garden";
+const chai = require("chai");
+const assert = chai.assert;
+
+describe("Garden tests", function () {
+   let myGarden = new Garden();
+   it("should be able to create a Garden", function () {
+      assert.instanceOf(myGarden, Garden);
+   });
+});
+
+/* Great! Now we can group tests into test blocks with "describe"
+   Let's test against more requirements.
+   A garden should be able to add different kinds of plants. Let's start with Fruit Trees!
+   Import "FruitTree" from the "FruitTree.js" file
+   Declare a variable myFruitTree and assign it to a new FruitTree with "Strawberry" as a parameter inside of the "describe" block
+*/
```

Comment:
Again, great writing here. However we may need to split this up into two steps.

The first could read:

```
/* 
  Great! Now we can group tests into test blocks with `describe`.
  Let's test against more requirements.
  A garden should be able to add different kinds of plants. Let's start with Fruit Trees!
  Import `FruitTree` from the `FruitTree.js` file in `src`.
*/
```

And the next step, maybe called `garden-test008a.js`, could read:

```
// Inside the `describe` block, declare a variable `myFruitTree` and assign it to a new `FruitTree` with `"Strawberry"` as a parameter.
```